src/redux/saga/user.ts

```diff
@@ -0,0 +1,30 @@
+import { put, takeLatest } from 'redux-saga/effects'
+import axios from 'axios'
+
+import LocalStorage from '../../local-storage'
+import { LoginUserRequestAction } from '../types'
+
+import { loginUserSuccess, loginUserFail } from '../actions/user'
+
+function* loginUserSaga(action: LoginUserRequestAction) {
+  const email = yield action.payload.credential.email
+  const password = yield action.payload.credential.password
+  const history = yield action.payload.history
```

Comment:
I believe `yield` is only needed for function calls. Can you remove `yield` for these

Reply:
Sure.